static/css/components/buttonCta.less

```diff
@@ -127,7 +127,8 @@ a.cta-btn {
 }
 
 .waitinglist-message {
-  white-space: nowrap;
+  white-space: normal;
```

Comment:
```suggestion
```

Assuming `white-space: normal` is the default and so we can remove it, otherwise we can add back in w/ a followup

Reply:
Noted 👍